src/init.cpp

```diff
@@ -1092,8 +1096,11 @@ bool AppInit2(ThreadHandlerPtr threads)
         for (auto const& strFile : mapMultiArgs["-loadblock"])
         {
             FILE *file = fsbridge::fopen(strFile, "rb");
-            if (file)
+            if (file) {
                 LoadExternalBlockFile(file);
+
+                UpdateOutOfSyncByAge();
+            }
```

Comment:
It looks like this call is redundant since `LoadExternalBlockFile()` pushes blocks through the entire pipeline, so each block loaded will call `UpdateOutOfSyncByAge()`, right? I suppose it doesn't hurt, though...

Reply:
I think you are right. I will pull it.